optimizer/validator.go

```diff
@@ -113,7 +113,7 @@ func (v *validator) checkSameColumns(exprs ...ast.ExprNode) {
 	}
 }
 
-// checkFieldList checks there is only one '*" and each field has only one column,
+// checkFieldList checks there is only one '*" and each field has only one column.
```

Comment:
Grammar error.
checkFieldList checks if there is only one '*" and each field has only one column.